src/main/java/com/resend/core/util/ValidationUtil.java

```diff
@@ -23,5 +23,10 @@ public class ValidationUtil {
      * Error message for a null tag name.
      */
     public static final String TAG_NAME_NULL = "Tag name is mandatory and cannot be null";
+
+    /**
+     * Error message for a null domain name.
+     */
+    public static final String DOMAIN_NAME_NULL = "Domain name is mandatory and cannot be null";
```

Comment:
Is this the returned error message? If so, can we return the actual message the API return instead?

Reply:
Yep, that can be done. I can remove all validations. But It's generally a good practice to validate null fields in both the SDK and the API. By validating null fields in the SDK, you catch potential issues early in the development process and before making any API request.

Reply:
Yep, that can be done. I can remove all validations.